cspell.yml

```
@@ -21,6 +21,7 @@ words:
   - tatooine
   - zuck
   - zuckerberg
-  # Alternative spellings
-  - !implementor
-  - !implementors
+# Forbid Alternative spellings
+flagWords:
+  - implementor->implementer
+  - implementors->implementers  
```

I couldn't find this syntax in the documentation and confirmed it didn't work locally; but this simpler form did:

```suggestion
  - implementor
  - implementors
```